public/css/docs.min.css

```diff
@@ -1008,6 +1008,14 @@ body {
     .bd-brand-item h1 {
         font-size: 4rem
     }
+
+    .bd-brand-logos {
+      margin-bottom: 0;
+    }
+
+    .navbar {
```

Comment:
Could you please remove this directive. The one above should be just fine

Reply:
Hmm Looks like the line numbers are messed up.
I think @Niden means keep
```
.bd-brand-logos {
   margin-bottom: 0;
}
```
And remove the Navbar directive (otherwise it get's too big)

Reply:
Yep that one lol sorry.